blocks/api/serializer.js

```diff
@@ -177,6 +180,7 @@ export function getBeautifulContent( content ) {
  * @return {string} HTML.
  */
 export function getBlockContent( block ) {
+	// @todo why not getBlockInnerHtml?
```

Comment:
@aduth @youknowriad seems we should go with `getBlockInnerHtml` for consistency.

Reply:
Yes, I'd like to align more of the terminology here. This is a good example. Also from the parser output, I'd like to normalize `attrs` -> `attributes`, `blockName` -> `name`.

https://github.com/WordPress/gutenberg/blob/aa1df371e2e85f4ac06a40cbe37643e2c63a9601/blocks/api/parser.js#L186-L191